.circleci/config.yml

```diff
@@ -22,45 +22,38 @@ jobs:
   test-policies:
     machine: true
     environment:
-      KIND_VERSION: v0.10.0
-      KUBERNETES_VERSION: v1.19.7
+      KIND_VERSION: v0.11.1
+      KUBERNETES_VERSION: v1.21.1
```

Comment:
You'll want to update the kubernetes version also in the `.abs/main.yaml`

Reply:
Done.

Reply:
I am still looking for a way to have a single source of truth for this - currently thinking the make file / env and then replacing in `main.yaml`. 

Nothing seems beautiful though :sweat_smile: